accepted/PSR-12-extended-coding-style-guide-meta.md

```diff
@@ -209,3 +209,8 @@ _**Note:** Order descending chronologically._
 
 * [Inspiration Mailing List Thread](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!topic/php-fig/wh9avopSR9k)
 * [Initial Mailing List PSR Proposal Thread](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!topic/php-fig/MkFacLdfGso)
+
+# 9. Errata and clarifications
+
+ * The code example for anonymous classes has ben fixed (see [#1206](https://github.com/php-fig/fig-standards/pull/1206)) because it was wrong; omitting parentheses after the `class` keyword should not be allowed, since it would be in contrast with what is prescribed for normal classes.
```

Comment:
I suggest using "is" instead of "should", because the original rule is a "MUST", not a "SHOULD".

```suggestion
 * The code example for anonymous classes has ben fixed (see [#1206](https://github.com/php-fig/fig-standards/pull/1206)) because it was wrong; omitting parentheses after the `class` keyword is not allowed, since it would be in contrast with what is prescribed for normal classes.
```